pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -616,6 +616,23 @@ func readPostgresStmt(
 				if !found {
 					return colinfo.NewUndefinedColumnError(cmd.Column.String())
 				}
+			case *tree.AlterTableSetVisible:
```

Comment:
can you add a test case for this?

somewhere in https://github.com/cockroachdb/cockroach/blob/6e5dfc46eab59f7fd6e77ee5dbab5bac715330de/pkg/ccl/importccl/import_stmt_test.go#L5559 !